third_party/blink/public/platform/web_media_player.h

```diff
@@ -389,6 +393,13 @@ class WebMediaPlayer {
   // Adjusts the frame sink hierarchy for the media frame sink.
   virtual void RegisterFrameSinkHierarchy() {}
   virtual void UnregisterFrameSinkHierarchy() {}
+
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
```

Comment:
Combine this `USE_STARBOARD_MEDIA` block with `GetSetBoundsCB()`, either move this up or move `GetSetBoundsCB` down.